website/src/main/java/com/google/step/data/OrganizationInfo.java

```diff
@@ -65,4 +79,43 @@ public Query getQueryForDuplicates() {
   public static OrganizationInfo createInstanceFrom(HttpServletRequest request) {
     return new OrganizationInfo(getOrgEntityFrom(request));
   }
+
+  public static List<OrganizationInfo> getOrganizationsFrom(CSVReader csvReader) throws IOException {
+    List<OrganizationInfo> organizations = new ArrayList<>();
+    String[] nextRecord = new String[3];
+    int index = 0; 
+    try {
+      while ((nextRecord = csvReader.readNext()) != null) { 
```

Comment:
Important: I added a header in the CSV, so please skip the first line when reading processed data